core/platform/lf_linux_support.c

```diff
@@ -1,3 +1,4 @@
+#if !defined(PLATFORM_ARDUINO)
```

Comment:
It seems a little strange that this would be necessary. Clearly, if the platform is Linux, it isn't Arduino. Same issue in other support file as well. I bet this can be refactored so that this isn't needed.